src/wrapper/java_vm/vm.rs

```diff
@@ -166,6 +166,11 @@ impl JavaVM {
     }
 
     /// Create a JavaVM from a raw pointer.
+    ///
+    /// # Safety
+    ///
+    /// Assumes a valid pointer is retrieved from the `JNI_CreateJavaVM` JNI function. Only does a
```

Comment:
Is 'Assumes' a recommended wording for such sections (vs 'Expects' or 'Requires')?

Reply:
I think no. Its main purpose is to describe the safety preconditions for an unsafe function.